### DISCUSSION TITLE
Manual control greyed out, can't seem to pair with sensors either

### DISCUSSION THREAD

**Discussion by ultimaoNe** [Rem0o/FanControl.Releases#1425] [Q&A]
I do not seem to have option for "manual" control of fans. It is greyed out. 
Not sure how to pair fans to sensors either. I tried automatic and nothing changed.

I can do a linear curve or flat for my fans , thats not the issues.

Using Motherboard 
MSI X570-A PRO

**Comment by Rem0o** [comment]
Turn on the toggle switch first?

**Reply by ultimaoNe** [reply]
I just figured that out. Thought it was from BIOS update. 

Thats counter intuitive to have to turn on a curve option..then gain manual control. 

Should just be able to choose manual control without anything.
Or at least have a pop up when I mouse over manual control to say turn on a curve to gain access to manual.

Because in Jayztwocents video. You didn't have to click on the toggle for curve or not curve and just access the manual fan option. 

Thanks though.
